recipes/boost-ext-ut/all/conanfile.py

```diff
@@ -67,9 +67,7 @@ def validate(self):
     def config_options(self):
         if Version(self.version) <= "1.1.8":
             del self.options.disable_module
-
-    def configure(self):
-        if is_msvc(self) and "disable_module" in self.options.values:
+        elif is_msvc(self):
             self.options.disable_module = True
```

Comment:
So modules are disabled only for msvc, while currently msvc is the only compiler supporting modules? Doesn't make sense.

Reply:
Found the related issue: https://github.com/boost-ext/ut/pull/447#issue-906457259

Looks like this needs to be disabled on MSVC, so maybe the `configure` method is actually the best place for it?

Reply:
I would say self.config_options, then raise in Validate if it's False and msvc, since it might be fixed in future versions.

Reply:
I've added a check to the `validate()` method.